ProQuest.js

```diff
@@ -499,6 +517,61 @@ function getItemType(types) {
 	return guessType;
 }
 
+function scrapeEbraryResults(refs) {
+	// Since we can't chase URLs, let's get what we can from the page
+	
+	for(var i = 0; i < refs.length; i++) {
+		var ref = refs[i];
+		var hiddenData = ZU.xpathText(ref.html, './span');
+		var visibleData = ZU.xpathText(ref.html.getElementsByClassName('results_list_copy'), './text()').trim().split(';').reverse();
```

Comment:
No problem.

Also, the Array.prototype.map() call described above doesn't work. I've been fiddling with it in Scaffold and it keeps producing an error. Turns out getElementsByClassName() is returning an object, not an array or a string. Getting textContent from the returned object will also grab extra text we don't want because there's an <a> node at the same level as the text node, though we can split that on \n to separate the contents of the <a> text node out.